clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/artifact/ReplacerTest.java

```diff
@@ -28,35 +28,7 @@
 import com.netflix.spinnaker.clouddriver.kubernetes.description.manifest.KubernetesManifest;
 import com.netflix.spinnaker.kork.artifacts.model.Artifact;
 import io.kubernetes.client.openapi.JSON;
-import io.kubernetes.client.openapi.models.V1ConfigMapEnvSource;
-import io.kubernetes.client.openapi.models.V1ConfigMapEnvSourceBuilder;
-import io.kubernetes.client.openapi.models.V1ConfigMapKeySelector;
-import io.kubernetes.client.openapi.models.V1ConfigMapKeySelectorBuilder;
-import io.kubernetes.client.openapi.models.V1ConfigMapVolumeSource;
-import io.kubernetes.client.openapi.models.V1ConfigMapVolumeSourceBuilder;
-import io.kubernetes.client.openapi.models.V1Container;
-import io.kubernetes.client.openapi.models.V1ContainerBuilder;
-import io.kubernetes.client.openapi.models.V1CrossVersionObjectReferenceBuilder;
-import io.kubernetes.client.openapi.models.V1EnvFromSource;
-import io.kubernetes.client.openapi.models.V1EnvFromSourceBuilder;
-import io.kubernetes.client.openapi.models.V1EnvVar;
-import io.kubernetes.client.openapi.models.V1EnvVarBuilder;
-import io.kubernetes.client.openapi.models.V1EnvVarSource;
-import io.kubernetes.client.openapi.models.V1EnvVarSourceBuilder;
-import io.kubernetes.client.openapi.models.V1HorizontalPodAutoscaler;
-import io.kubernetes.client.openapi.models.V1HorizontalPodAutoscalerBuilder;
-import io.kubernetes.client.openapi.models.V1Pod;
-import io.kubernetes.client.openapi.models.V1PodBuilder;
-import io.kubernetes.client.openapi.models.V1ReplicaSet;
-import io.kubernetes.client.openapi.models.V1ReplicaSetBuilder;
-import io.kubernetes.client.openapi.models.V1SecretEnvSource;
-import io.kubernetes.client.openapi.models.V1SecretEnvSourceBuilder;
-import io.kubernetes.client.openapi.models.V1SecretKeySelector;
-import io.kubernetes.client.openapi.models.V1SecretKeySelectorBuilder;
-import io.kubernetes.client.openapi.models.V1SecretVolumeSource;
-import io.kubernetes.client.openapi.models.V1SecretVolumeSourceBuilder;
-import io.kubernetes.client.openapi.models.V1Volume;
-import io.kubernetes.client.openapi.models.V1VolumeBuilder;
+import io.kubernetes.client.openapi.models.*;
```

Comment:
Ah, IntelliJ did this on its own accord, happy to revert to specifics.

Reply:
No worries; in general I think we prefer specific imports, but as our code formatter doesn't enforce it I don't usually comment either way on PRs.  (And in this case it is replacing *a lot* of imports.)